src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -13564,26 +13546,6 @@ public static int Main()
                 new ErrorDescription { Code = (int)ErrorCode.ERR_PartialMethodOnlyInPartialClass, Line = 5, Column = 18 });
         }
 
-        [Fact]
-        public void CS0752ERR_PartialMethodCannotHaveOutParameters()
```

Comment:
>CS0752ERR_PartialMethodCannotHaveOutParameters [](start = 20, length = 46)

Let's keep the test, even though the behavior has changed.

Reply:
Restored the test.

Reply:
Thanks. It looks like the order of this method and the previous method has changed. Please consider reverting the order.

---
In reply to: [418715785](https://github.com/dotnet/roslyn/pull/43582#discussion_r418715785) [](ancestors = 418715785)

Reply:
Thanks, I figured out how to easily compare my working copy to the base, now it's easier to spot these kinds of mixups.